policies/policy-change-process.md

```diff
@@ -1,8 +1,11 @@
 Policy on Proposing General Policy Changes
 ==========================================
 
-This policy represents the way that any additions or changes to the existing
-policies are proposed, edited, finalized, and approved.
+This policy represents the way that any additions or changes to the
+existing policies are proposed, edited, finalized, and approved.
```

Comment:
I understand there's history behind this edit...  but it looks like an unnecessary rewrap at this point :wink: